dj_search/search.py

```diff
@@ -97,7 +97,7 @@ def _do_search(self, level=None):
                 continue
 
             # extract the table this matched string belongs to
-            for class_start in re.finditer(' *class', self._definition_string[:match.span()[-1]]):
+            for class_start in re.finditer(' *class ', self._definition_string[:match.span()[-1]]):
```

Comment:
@ttngu207 Would you elaborate on what bug/issue that this is supposed to fix? Thinking I am missing some context here.

Reply:
ah yes, this is to handle a case to search for the start of a DataJoint table class, e.g. starting with `class Tablename...` (need that space between `class` and the class name)
Was a bug before, as it would've matched string like `classifier` for example

Reply:
Perhaps look only for patterns at the beginning of a line, i.e. `"^\s*class\s"`.